README.md

```diff
@@ -165,6 +165,28 @@ formFields | Map or `Array<Pair>` | no | Additional form fields for multipart ht
 
 Base64-encoded content of the recorded media file if `remotePath` parameter is falsy or an empty string.
 
+### windows: deleteFile
+
+Remove the file from the file system. This feature is potentially insecure and thus needs to be
+explicitly enabled when executing the server by providing `delete_files` key to the list
+of enabled insecure features. Refer https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/security.md for more details.
```

Comment:
It is easier to read when the actual url is presented as a link, for example `Refer to [Appium Security document](https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/security.md) ...`

Reply:
same below

Reply:
Fixed.